docs/logging/03-03-configure-logcollector.md

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
can we change the name of the doc to `08-01-configure-logcollector.md` ? So that it's a tutorial :)